SonarQube.Bootstrapper/ArgumentProcessor.cs

```diff
@@ -107,11 +115,11 @@ public static bool TryProcessArgs(string[] commandLineArgs, ILogger logger, out
 
                 if (phase == AnalysisPhase.PreProcessing)
                 {
-                    settings = CreatePreProcessorSettings(baseChildArgs, properties, globalFileProperties, logger);
+                    settings = CreatePreProcessorSettings(baseChildArgs, properties, globalFileProperties, installLoaderTargets, logger);
                 }
                 else
                 {
-                    settings = CreatePostProcessorSettings(baseChildArgs, properties, logger);
+                    settings = CreatePostProcessorSettings(baseChildArgs, properties, installLoaderTargets, logger);
```

Comment:
The post processor has nothing to do with this new boolean setting but I figured it it better to keep the bootstrappersettings object in a good state